client/views/actions/whois.tpl

```diff
@@ -38,8 +38,10 @@
 {{/if}}
 
 {{#if whois.special}}
-	<dt>Special:</dt>
-	<dd>{{whois.special}}</dd>
+	{{#each whois.special}}
+		<dt>Special:</dt>
+		<dd>{{this}}</dd>
+	{{/each}}
```

Comment:
So this would produce something like:

```
Special: Foo
Special: Bar
Special: Baz
```

Right?

What about comma-separated or something?

```
Special: Foo, Bar, Baz
```

Note that I don't know what usually a special value so maybe this makes no sense. Maybe splitting with `br`s works better if those are sentences, or already comma-separated values.

Reply:
Meh, displaying it multiple times should be fine and less confusing for those that actually see multiple values (which should be rare in some specific cases).

Reply:
That's fair.